packages/cli/src/commands/ci-check.js

```diff
@@ -34,19 +42,21 @@ async function main() {
     clearInterval(interval);
     console.log();
     reporter.success('Done! 🎉');
-  } finally {
+  } catch (error) {
     clearInterval(interval);
+    console.log();
```

Comment:
just wondering what does `console.log` with no arguments do here?

Reply:
@emyarod just adds a separating `\n` in the output from the `...`'s written by the interval 👍